config/application.rb

```diff
@@ -81,22 +81,6 @@ class Application < Rails::Application
       ]
     end
 
-    # Settings dependent on locale
-    #
-    # We need to set this config before the promo environment gets loaded and
-    # after the spree environment gets loaded...
-    # This is because Spree uses `Spree::Config` while evaluating classes :scream:
-    #
-    # https://github.com/spree/spree/blob/2-0-stable/core/app/models/spree/calculator/per_item.rb#L6
-    #
-    # TODO: move back to spree initializer once we upgrade to a more recent version
-    #       of Spree
-    initializer 'ofn.spree_locale_settings', before: 'spree.promo.environment' do |app|
-      Rails.application.reloader.to_prepare do
-        Spree::Config['currency'] = ENV['CURRENCY']
-      end
-    end
-
```

Comment:
This diff doesn't show CURRENCY used anywhere in the app, so I had a look and found that it already refers directly to the ENV var, hence no further change needed 👍